src/Generation/ResourcesGenerator.php

```diff
@@ -216,7 +216,7 @@ public static function generateClientConfig(
                     $retryParams = $retryParams->append([
                         $paramsName, Vector::new([
                             ['initial_retry_delay_millis', is_null($policy) ? 0 : $durationToMillis($policy->getInitialBackoff())],
-                            ['retry_delay_multiplier', is_null($policy) ? 0 : $policy->getBackoffMultiplier()],
+                            ['retry_delay_multiplier', is_null($policy) ? 0 : round($policy->getBackoffMultiplier(), 1)],
```

Comment:
1dp is possibly a little too inaccurate? I could imagine users wanting to specify maybe 1.25, or 1.3333? Maybe use 5 or 6 dp?

Slightly curious as to why I don't see this on my machine, or on github actions...

Reply:
Done.